META-INF/plugin.xml

```diff
@@ -584,6 +584,11 @@ Bug-fix release, fixed a new bunch of false-positives<br/>
         shortName="ParameterDefaultValueIsNotNullInspection"      displayName="Non-null parameters default value"
         groupName="Code Style"                                    enabledByDefault="false" level="WEAK WARNING"
         implementationClass="com.kalessil.phpStorm.phpInspectionsEA.inspectors.codeSmell.ParameterDefaultValueIsNotNullInspector"/>
+    <localInspection language="PHP" groupPath="PHP,Php Inspections (EA Extended)"
+        shortName="DisallowWritingIntoStaticPropertiesInspection" displayName="Disallow writing into static properties"
+        groupName="Code Style" enabledByDefault="true" level="WARNING"
```

Comment:
@funivan: can we release it as optional (`enabledByDefault="false"`), later we can rename it and activate by default. I'd merge the PR immediately then/

The background is to make bug-fix and QF release - to support the crowdfunding campaign PR.